core/templates/components/skill-mastery/skill-mastery.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="oppia-skill-mastery-viewer" *ngIf="!loadingMessage">
+  <div class="skill-mastery-degree">
+    <b> Mastery Degree: </b>
+    <div style="margin-bottom: 10px">
+      {{ getSkillMasteryPercentage() }} %
```

Comment:
```suggestion
      {{ getSkillMasteryPercentage() }}%
```

There wasn't an extra space between the interpolation and %.

Reply:
Done